verification/curator-service/ui/src/components/App.tsx

```diff
@@ -216,7 +241,53 @@ class App extends React.Component<Props, State> {
                         }}
                     >
                         <div className={classes.drawerHeader}></div>
-                        <Divider />
+                        {this.hasAnyRole(['curator']) && (
```

Comment:
Should this be a separate component?

Reply:
Perhaps we can pull it out if we use another menu button somewhere else in the UI in the future